tool-wrapper.inc

```diff
@@ -119,7 +119,6 @@ if [ ! -s $LOG.ok ] ; then
   exit 1
 fi
 
-eval `tail -n 1 $LOG.ok`
```

Comment:
That is used by cbmc.inc: `echo "EC=$EC"` - as the final result needs to be communicated back. Either 2LS should be doing the same, or, better, move this into the run function of cbmc.inc.